### DISCUSSION TITLE
How get I get a user pofile

### DISCUSSION THREAD

**Discussion by jacksonsophat** [pocketbase/pocketbase#2056] [Q&A]
Here's an example of the code: 

`const record = await pb.collection('users').getFirstListItem('username="jackson"');`

I think it might be API Rules related, but I'm not sure. 
Thank you

**Comment by ganigeorgiev** [comment]
The above filter should work.

Make sure that your client request satisfies the List/Search API rule of the `users` collection (you can temporary make it empty string to grant everyone permission and see if that's causing the issue).
